onnxruntime/test/framework/function_test.cc

```diff
@@ -521,5 +534,56 @@ TEST(FunctionTest, ConstantFoldingInSubGraph) {
   Check(code, "X", {1.0, 2.0, 3.0}, "Y", {3.0, 4.0, 5.0, 3.0, 4.0, 5.0, 3.0, 4.0, 5.0});
 }
 
+TEST(FunctionTest, TestInlinedLocalFunctionRemoved) {
```

Comment:
Can we add a test or test with some models that have been recently exported which have a decent number of nodes and multiple levels?

Has the code coverage been checked to ensure any subgraph handling is being exercised by tests?

Reply:
The test coverage is good with these two tests, as they run both branches of the code. The models I used to test the code are 100Mb 500Mb in size. These are the representative models given by the converter team.

Reply:
The models I used for testing and coverage are too big for the source control (100mb and 500MB). They have plenty of nodes, subgraphs. The test models in unit tests also have subgraphs.

I just ran the test coverage.